cluster-configuration/services-configuration.yaml

```diff
@@ -34,27 +34,27 @@ cluster:
   docker-registry-info:
 
     # If public, please fill it the same as your username
-    docker-namespace: your_registry_namespace
+    docker-namespace: openpai
 
     # E.g., gcr.io. If public，fill docker_registry_domain with word "public"
     # docker_registry_domain: public
-    docker-registry-domain: your_registry_domain
+    docker-registry-domain: docker.io
     # If the docker registry doesn't require authentication, please leave docker_username and docker_password empty
-    docker-username: your_registry_username
-    docker-password: your_registry_password
+    docker-username: 
+    docker-password: 
 
-    docker-tag: your_image_tag
+    docker-tag: latest
 
     # The name of the secret in kubernetes will be created in your cluster
     # Must be lower case, e.g., regsecret.
-    secret-name: your_secret_name
+    secret-name: regsecret
 
 
 hadoop:
   # If custom_hadoop_binary_path is None, script will download a standard version of hadoop binary for you
   # hadoop-version
   # http://archive.apache.org/dist/hadoop/common/hadoop-2.7.2/hadoop-2.7.2.tar.gz
-  custom-hadoop-binary-path: None
+  custom-hadoop-binary-path: ./customhadoop/
```

Comment:
>./customhadoop/ [](start = 29, length = 15)

Maybe we can keep this value empty as using custom hadoop binary path is optional.

Reply:
due to some recent change in rest-server, we lose the compatibility to standard hadoop. I would even suggest we remove the description about standard hadoop for now.

---
In reply to: [195048031](https://github.com/Microsoft/pai/pull/727#discussion_r195048031) [](ancestors = 195048031)

Reply:
Is the "./customhadoop" our default hadoop-ai built location?

---
In reply to: [195050624](https://github.com/Microsoft/pai/pull/727#discussion_r195050624) [](ancestors = 195050624,195048031)

Reply:
that is the issue: we don't have a default hadoop-ai build location. user will be confused, we need to specify one.

---
In reply to: [195295594](https://github.com/Microsoft/pai/pull/727#discussion_r195295594) [](ancestors = 195295594,195050624,195048031)

Reply:
Agree with the idea that we remove the description about the standard Hadoop in comment.

Reply:
change to /pathHadoop/ to be consistent with the description in set-up-dev-box"

---
In reply to: [195303519](https://github.com/Microsoft/pai/pull/727#discussion_r195303519) [](ancestors = 195303519,195295594,195050624,195048031)